### ISSUE TITLE
ZeroNet and Emercoin blockchain domain zones

### ISSUE THREAD

**Issue by mechnotech** [HelloZeroNet/ZeroNet#695]
Dear ZeroNet team, 

Greetings from Emercoin team

We like very much the idea of ZeroNet and especially the implementation. 
We would be very happy if ZeroNet could support Emercoin blockchain domain zones - * .lib, coin, emc, bazar.

We have many interesting features and related services that could be useful to you. 
For example blockchain-based magnet links and many others.
We would be happy to collaborate with you.

Waiting for your answer, 
Sincerely, Eugene Shumilov
Emercoin CEO

mechnotech@emercoin.com

**Comment by iskradelta** [comment]
Emercoin has Proof-of-Stake, does it also have the centralized checkpointing component?

**Comment by s0r00t** [comment]
Duplicate of #142

**Comment by castamonte** [comment]
emercoin seems more perspective than other coins

**Comment by mechnotech** [comment]
> Emercoin has Proof-of-Stake, does it also have the centralized checkpointing component?

We have decentralized ones. But in the next release we will reject soft checkpoint at all.

**Comment by iskradelta** [comment]
@mechnotech How light-weight is the emercoin domain-name/identity client? Namecoin does suffer a bit from that, the full node is too fat, and the thin clients are not yet existent, or suffer from trusting a central server too much.

I was looking more into byteball to build a namesystem on that, as it looks the light-weight client will be easier to build there and more secure/more-decentralized.

**Comment by maxihatop** [comment]
> How light-weight is the emercoin domain-name/identity client?

Currently, it consumes ~170MB RAM and ~300MB HDD. Since blockchain nature, HDD part will be increased over time. Anyway, it enough compact to run on Raspberry Pi hardware, and we support this platform.
Important distinctive feature - our node can process native DNS requests RFC1035, so do not needed any external DNS software, if you will use EMC domains only.

Also, if you would like, yo can forward your DNS-requests to the public EMC DNS gateway, emcdns.emercoin.com. In this case, you will do not needed run any client on your side.
Of course, if you would like, you can create your own gateway(s), out of our control.

**Comment by JeremyRand** [comment]
> Namecoin does suffer a bit from that, the full node is too fat, and the thin clients are not yet existent, or suffer from trusting a central server too much.

That is incorrect; Namecoin has an SPV client (based on BitcoinJ/libdohj) that does not use any central servers (unless you're referring to DNS seeds, which are only used for network bootstrapping).  @cassiniNMC talked to Tamas about this months ago.

As an aside, it is an intentional design decision of Namecoin to not process the DNS protocol in Namecoin Core, because doing so would make sandboxing substantially less effective, which would be damaging to security.  I'm curious whether Emercoin ever considered that problem when they designed their system.

**Comment by DaniellMesquita** [comment]
Reference:
https://github.com/HelloZeroNet/ZeroNet/issues/558
